.travis.yml

```diff
@@ -78,7 +87,7 @@ jobs:
           on:
             repo: angular/angular.js
             all_branches: true
-            condition: $DEPLOY_DOCS
+            condition: "$DEPLOY_DOCS == true"
```

Comment:
Are you sure this works? (I would expect this to always evaluate to true).
Maybe `condition: env(DEPLOY_DOCS) = true`?